adapters/BasicAdapter/src/main/kotlin/me/gabber235/typewriter/entries/event/RunCommandEventEntry.kt

```diff
@@ -12,7 +12,7 @@ class RunCommandEventEntry(
 	override val id: String = "",
 	override val name: String = "",
 	override val triggers: List<String> = emptyList(),
-	val command: String = "",
+	override val command: String = "",
 ) : EventEntry
```

Comment:
This entry is specific to look for existing commands and seeing if the player has ran them. As such it allows for regex which will conflict with custom commands.
I think it is best to make a new entry which registers a command. It is easier, and makes the distinction more clear.

Reply:
I know how to implement this, but it seems simpler for the user to just put the on command run entry and it be registered. Plus, where would the command register entry go in the page? After the command is run?

Reply:
I don't fully grasp what you mean. The way I see it is there are two use cases for commands.

1. You want a custom command which will be registered and is specific for the use case. An example of this would be `/sing` which may play a sound and sends a message.

2. You want to hook in to a command and see if it has been ran. An example of this would be `(faction|f) create`. Which checks if the users runs `/faction create`. This command already exists and we just want some additional logic ran by Typewriter.

I think both are valid use cases but I imagine making two entries makes the case distinction easier for the user.

Reply:
I see what you mean. I am having trouble getting it to register in the first place, and in my code it isn't even registering until the server restarts. The code in this PR was working, but as soon as I changed it with some of the things you mentioned, it doesn't work. Any advice?

Reply:
Maybe make the changes and then I'll have a look at it tomorrow to see why it is not working.

Reply:
Ok, thanks